build.gradle

```diff
@@ -48,6 +48,14 @@ dependencies {
     //compile 'io.grpc:grpc-protobuf:1.+'
 }
 
+task runApp(type:JavaExec) {
```

Comment:
Pls rename it to `cli` or something like that

Reply:
Just cli? :)

Reply:
`gradle cli -Dexec.args="-h"` looks pretty neat :)
I don't mind some other name, though, just don't want meaningless "runapp"

Reply:
Done